### PR TITLE
🌎Added WorldHideaway Presence

### LINKED ISSUE #3101: WorldHideAway | worldhideaway.com
**Prerequisites and essential questions** 
- [x] Is it a popular site?
- [x] Is the website older than 2 months? 
- [ ] Is the site locked to a specific country/region?
- [ ] Is the site a paid service? (e.g. Netflix, Hulu)
- [ ] Does the website feature NSFW content? (e.g. porn, etc...)
- [ ] Are you a donator/patron?
- [x] Do you acknowledge that coding presences is completely voluntary and may take time for your service to be added regardless of priority?

**What's your Discord username?** 
srjoel#0251

**What's the name of the service?** 
www.worldhideaway.com

**What should the Presence display?** 
The different sections of web (Guides section, news, home)

**If possible, please provide a logo for the service (512x512 minimum)** 
Sin título-1_Mesa de trabajo 1

### PR DESCRIPTION
**Pull request by DarkVillager**
Resolves #3101 
Added Full Page Functionality.
wh1
wh2
wh3
wh4
wh5
wh6

### CODE REVIEW

File: websites/W/Worldhideaway/presence.ts [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "logo"
+  },
+  page = window.location.pathname,
+  expage = window.location.href,
+  host = document.location.hostname; 
+
+    presenceData.startTimestamp = browsingStamp;
+    if (host === "www.worldhideaway.com" || host === "worldhideaway.com") {
```
Comment (Bas950) [inline websites/W/Worldhideaway/presence.ts:16]: useless if case...

File: websites/W/Worldhideaway/presence.ts [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    presenceData.state = `Gestos`;
+  }
+  if (page === "/stickers/") {
+    presenceData.details = "Viendo:";
+    presenceData.state = `Pegatinas`;
+  }
+  if (page === "/salas/") {
+    presenceData.details = "Viendo:";
+    presenceData.state = `Salas`;
+  }
```
Comment (Bas950) [inline websites/W/Worldhideaway/presence.ts:104]: `else if` ???

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/W/Worldhideaway/presence.ts (hunk 2 above)
Comment (DarkVillager) [reply websites/W/Worldhideaway/presence.ts:104]: ?

Comment (Bas950) [reply websites/W/Worldhideaway/presence.ts:104]: instead of constantly using new ifs just use an else if?

**Review by Bas950 — CHANGES_REQUESTED** [review]
yea this doesnt look right...

**Review by StrikerFRFX — CHANGES_REQUESTED** [review]
Did you have conflicts to resolve on the `presence.ts` file? Looks like you fucked it....

**Comment by DarkVillager** [comment]
> Did you have conflicts to resolve on the `presence.ts` file? Looks like you fucked it....

Will fix dw

**Pull request closed without merging**